nano/node/blockprocessor.cpp

```diff
@@ -71,6 +71,7 @@ void nano::block_processor::add (std::shared_ptr<nano::block> block_a, uint64_t
 void nano::block_processor::add (nano::unchecked_info const & info_a)
 {
 	debug_assert (!nano::work_validate (*info_a.block));
+	if (info_a.block->difficulty () >= nano::work_threshold (info_a.block->work_version ()))
```

Comment:
This line is temporary until the next PR where the check will be in ledger processing, separately for each block variant.